scenarios/kubernetes_build.py

```diff
@@ -30,8 +31,55 @@ def check(*cmd):
     print >>sys.stderr, 'Run:', cmd
     subprocess.check_call(cmd)
 
+def check_no_stdout(*cmd):
+    """Log and run the command, suppress stdout, raising on errors."""
```

Comment:
I think we want to suppress stderr? otherwise we'll still get the misleading "not found" error message.

Reply:
done for both now